src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/MethodLength/context/Test.java

```diff
@@ -0,0 +1,8 @@
+public class Test {
+    private String name;
+
+    public void test() { // violation context, should be suppressed after patching
```

Comment:
Test.java is always after patch.

> should be suppressed after patching

we will see this in expected.txt, lets not duplicate information.
In Input files we keep comment on lines where violation is present without PatchFilter.

Reply:
please make it simple `// violation` to show clearly that is not related to any strategy in filter.

to avoid any other confusion we might have do ` // violation without filter` to make it absolutely clear. If you agree lets update all Inputs (in separate issue).